docs/detections/session-view.asciidoc

```diff
@@ -84,4 +89,23 @@ Session View includes an additional badge not pictured above:
 * The *Exec user change* badge highlights exec user changes, such as when a user escalates to root:
 +
 [role="screenshot"]
-image::images/session-view-exec-user-change-badge.png[The Exec user change button,width=80%,height=80%]
+image::images/session-view-exec-user-change-badge.png[The Exec user change badge,width=80%,height=80%]
+
+* The *Output* badge appears next to commands which generated TTY output. Click it to open Terminal output view.
++
+[role="screenshot"]
+image::images/session-view-output-badge.png[The Output badge,width=80%,height=30%]
+
+[[session-view-output]]
+[discrete]
+=== Terminal output view UI
+Terminal output view can help provide a more complete understanding of commands executed by users or adversaries, and assist with auditing and compliance objectives. It allows you to review a session's TTY output, in the context of the commands which generated it. It includes both TTY output, and commands entered into a shell as typed — provided the shell was in echo mode (for background information about Linux terminals, refer to https://github.com/elastic/security-team/blob/main/docs/adaptive-workload-protection-team/architecture/linux-terminals.md[Linux terminals]).
```

Comment:
security-team repo is private, @qcorporation @m-sample do we know of a good place to put this material for public viewing?

Reply:
Yeah, security-team and docs.elastic.co are internal resources. If we want to include this as a public-facing doc then we're going to have to pull in linux-terminals.md into the docs repo
@benironside is your intention to making linux-terminals.md public facing? If so then we'll need to pull that out of the private repo

Reply:
@qcorporation I'd like to be able to link to it, IF the team feels it's appropriate to make it public facing. The background info seems relevant here.

Reply:
> @m-sample can we make this public? (is it ready?) https://github.com/elastic/security-team/blob/main/docs/adaptive-workload-protection-team/architecture/linux-terminals.md

It will need a clean up to make it broadly consumable.  It is also very terse and assumes the reader has a very good understanding of Linux (e.g. what is a process group, signals etc).  That said I think we *should* share some technical outline of how TTY's work to help customers really understand this feature.

Reply:
Unless anyone objects and suggests another resource, I'll remove it for now, and create a docs issue for adding a link with background info there.